ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-main.yaml

```diff
@@ -0,0 +1,75 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.19-openshift-4.14:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.19-openshift-4.14
+build_root:
+  from_repository: true
+images:
+- dockerfile_literal: |-
```

Comment:
why isn't the dockerfile in the repo?

Reply:
If we are diverging from the majority, we'll switch to in-repo

Reply:
yeah a quick grep shows me a handful of `dockerfile_literal` usage, but not a ton(relatively speaking) and i don't think any of them are in the payload